RawWaterManagement/HydraulicSimulationScenario/examples/example-normalized.jsonld

```diff
@@ -0,0 +1,144 @@
+  },
+  "alternateName": {
+    "type": "Property",
+    "value": "So product organization last."
+  },
+  "description": {
+    "type": "Property",
+    "value": "Interview other plant pattern close policy focus. Well hotel goal ball suddenly third goal."
+  },
+  "dataProvider": {
+    "type": "Property",
+    "value": "Heart may put room. Dog product hot reflect expect. Write modern spring industry on summer."
+  },
+  "owner": {
+    "type": "Property",
+    "value": [
+      "urn:ngsi-ld:dataModel:items:QJVR:94419439",
+      "urn:ngsi-ld:dataModel:items:WDTQ:88171012"
+    ]
+  },
+  "seeAlso": {
+    "type": "Property",
+    "value": [
+      "urn:ngsi-ld:dataModel:items:JDRW:02822125",
+      "urn:ngsi-ld:dataModel:items:HSTA:26679121"
+    ]
+  },
+  "location": {
+    "type": "Property",
+    "value": {
+      "type": "Point",
+      "coordinates": [
+        -52.853197,
+        -113.959231
+      ]
+    }
+  },
+  "address": {
+    "type": "Property",
+    "value": {
+      "streetAddress": "Beat capital real decade agreement issue. Person hundred require answer image could.",
+      "addressLocality": "Generation couple expect environmental simply gas social. While expert my bed thus officer special.",
+      "addressRegion": "Wind want single message. Tend with return often parent. Including effect total sign could industry lay.",
+      "addressCountry": "Toward continue soon treatment. Tree lead spend color base marriage nature. Author statement or until.",
+      "postalCode": "Try experience huge. Mr tax wind all force second once. Money vote positive Democrat.",
+      "postOfficeBoxNumber": "Story account tax admit billion card find. Student sing never true start significant around.",
+      "areaServed": "Television human prepare. Customer likely not firm until."
+    }
+  },
+  "areaServed": {
+    "type": "Property",
+    "value": "Represent girl among hotel figure. Big new those politics trip."
+  },
+  "type": "HydraulicSimulationScenario",
+  "hasInputOpenChannelSystem": {
+    "type": "Property",
+    "value": "urn:ngsi-ld:dataModel:hasInputOpenChannelSystem:RCDK:47499254"
+  },
+  "hasSimulationResult": {
+    "type": "Property",
+    "value": "urn:ngsi-ld:dataModel:hasSimulationResult:XRNY:02688357"
+  },
+  "targetValues": {
+    "type": "Property",
+    "value": [
+      {
+        "waterAttribute": 864.6,
+        "targetURI": "urn:ngsi-ld:dataModel:targetURI:JXFD:60487647"
+      },
+      {
+        "waterAttribute": 864.6,
+        "targetURI": "urn:ngsi-ld:dataModel:targetURI:JXFD:60487647"
+      }
+    ]
+  },
+  "inputParameters": {
+    "type": "Property",
+    "value": [
+      "urn:ngsi-ld:dataModel:items:CBWI:21948924",
+      "urn:ngsi-ld:dataModel:items:MWGU:81565938"
+    ]
+  },
+  "createdBy": {
+    "type": "Property",
+    "value": "Behavior style report. Ability management test during foot that course nothing. Sound central myself before year."
+  },
+  "duration": {
```

Comment:
Use of unitcode is encouraged to ensure consistency of collected data. Otherwise, no one knows if e.g. a temperature is expressed in °C or K 
The specification recommends to take it from UN CEFACT (https://unece.org/DAM/cefact/recommendations/rec20/rec20.zip)